index.html

```diff
@@ -17,11 +17,11 @@
                 </p>
             </div>
         </section>
-		
+
 		<!-- deliverables -->
 		<section class="bg-light pb-5">
                   <div class="container">
-                      <h2 class="no-anchor text-center mb-3"> Resources </h2>
+                      <h2 class="no-anchor text-center mb-3"> Information about data management planning </h2>

```

Comment:
```suggestion
                      <h2 class="no-anchor text-center mb-3"> Further information </h2>

```
The whole page is about RDM or not?